control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -911,6 +911,16 @@ func (r *HostedControlPlaneReconciler) reconcile(ctx context.Context, hostedCont
 		return fmt.Errorf("failed to reconcile cluster version operator: %w", err)
 	}
 
+	// Block until cluster version operator is fully ready to ensure readiness before HCCO
```

Comment:
If you are using the availability prober, why do you need to block here? Isn't the `WaitForClusterVersionUpdated` param of the hcco enough?

Reply:
Yup, removed that block to avoid redundancy. Thanks, Cesar!